src/main/game/Game.java

```diff
@@ -48,7 +48,7 @@ private boolean gameIsNotOver() {
     }
 
     private void displayBoard() {
-        List<List<String>> activeBoard = this.board.rows();
+        List<Mark> activeBoard = this.board.grid;
```

Comment:
Instead of accessing the field directly, I would suggest making the `grid` field private and provide a "getter" method in order to access it.